src/README.md

```diff
@@ -0,0 +1,13 @@
+# Gaffr Source Folder
+
+This directory is for source files
+
+## Format
+
+There are lots of ways to organize source files. For now, let's keep things simple. We can refactor later.
+
+- Regular C++ files:
+    - `src/*.cpp`
+    - `src/*.h`
```

Comment:
Do we want this in an include directory instead?

Reply:
IMO it's only necessary when you have headers you want to expose to other projects so they can link against yours. Like, if you're publishing a `*.so`. We can always refactor later. I like to do that around the time the code base develops a more complex hierarchy.

Reply:
Sounds good, was just asking